src/app/related-search/related-search.component.html

```diff
@@ -1,7 +1,17 @@
 <div *ngIf="results?.length > 0" class="card">
     <h3>Searches related to {{this.keyword}}</h3>
     <div *ngFor="let result of results" class="related">
-      <a [routerLink]="resultsearch" [queryParams]="{query: result.label}" class="rellink">{{result.label}}</a>
+      <a [routerLink]="resultsearch" [queryParams]="{query: result.label,
+                                                      verify: false,
+                                                      nav: 'filetype,protocol,hosts,authors,collections,namespace,topics,date',
+                                                      start: 0,
+                                                      indexof: 'off',
+                                                      meanCount: '5',
+                                                      resource: 'global',
+                                                      prefermaskfilter: '',
+                                                      rows: 10,
```

Comment:
why don't you write a function which takes input `resultLabel` and outputs `queryParams` object?
where you could now call that function inside [queryParams]="somefunction(result.label)".
if this doesn't work implement similar to `docclick()`.